tools/rpm2img

```diff
@@ -320,20 +340,21 @@ fi
 cat <<EOF > "${BOOT_MOUNT}/grub/grub.cfg"
 set default="0"
 set timeout="0"
+set dm_verity_root="${DM_VERITY_ROOT[@]}"
```

Comment:
Any time a new command is used in the GRUB config, it's worth extra scrutiny since GRUB doesn't get updated by in-place updates. It'd be easy to rely on new functionality that would create problems on upgrade for older images.

Here I'm confident it's safe because we've used `set` commands forever, and never added a new GRUB module for that functionality.